app/assets/javascripts/discourse/tests/integration/components/file-size-input-test.js

```diff
@@ -0,0 +1,84 @@
+import { click, fillIn, render } from "@ember/test-helpers";
+import { hbs } from "ember-cli-htmlbars";
+import { module, test } from "qunit";
+import { setupRenderingTest } from "discourse/tests/helpers/component-test";
+import selectKit from "discourse/tests/helpers/select-kit-helper";
+
+module("Integration | Component | file-size-input", function (hooks) {
+  setupRenderingTest(hooks);
+
+  test("file size unit selector", async function (assert) {
+    this.set("value", 4096);
+    this.set("max", 8192);
+    this.set("onChangeSize", function () {});
+    this.set("updateValidationMessage", function () {});
+
+    await render(hbs`
+      <FileSizeInput
+        @sizeValueKB={{readonly this.value}}
+        @classNames="file-size-input-test"
```

Comment:
you can use a regular html attribute:

```suggestion
        class="file-size-input-test"
```